modules/rpc-util.js

```diff
@@ -0,0 +1,34 @@
+const request = require('request');
+
+class RpcRequest {
+
+    static fetchBalance(provider, accountHash) {
+        return new Promise(function(resolve, reject) {
+            var URL = `${provider}/chains/main/blocks/head/context/contracts/${accountHash}/balance`;
+
+            request(URL, function (error, response, body) {
+                if(error) {
+                    reject(error);
+                } else {
+                    resolve(body);
```

Comment:
This is JSON resonse. I would advice to parse the body here and then return JSON response.